src/legacy/ui/public/courier/fetch/__tests__/call_client.js

```diff
@@ -45,7 +45,9 @@ describe('callClient', () => {
     const { source: overrideSource, ...rest } = overrides;
 
     const source = {
-      _flatten: () => ({}),
+      _flatten: () => Promise.resolve({
```

Comment:
In general these changes are mostly to get the tests passing for this PR, they are completely replaced anyway in this PR: https://github.com/elastic/kibana/pull/45235